compiler/visq-unittest/test/testPalette.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
```

Comment:
maybe 2023? or set other files to 2022?

Reply:
Files in this PR are not newly created but moved from `visq`. So the copyright year is different file by file :)